src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberMethodSymbol.cs

```diff
@@ -24,15 +24,15 @@ protected struct Flags
         {
             // We currently pack everything into a 32 bit int with the following layout:
             //
-            // |              |a|b|e|n|vvv|yy|s|r|q|z|kk|wwwww|
+            // |               |a|b|e|n|vvv|y|s|r|q|z|kk|wwwww|
```

Comment:
>y [](http://example.com/codeflow?start=44&length=1)

Alternatively, rather than removing the bit, we could adjust the code to properly initialize it. I.e. make sure that it is not set from the beginning for scenarios that require delayed binding for return type. I think that is how it worked from the beginning, and then it got "lost" in one of the refactorings. From the code maintenance point of view, it is better to keep the flag and add an assert for an unexpected attempt to access value that is not ready.